app/src/main/java/free/rm/skytube/businessobjects/Sponsorblock/SBVideoInfo.java

```diff
@@ -0,0 +1,46 @@
+package free.rm.skytube.businessobjects.Sponsorblock;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SBVideoInfo {
+    protected boolean loaded = false;
+    protected double videoDuration;
+    protected List<SBSegment> segments = new ArrayList<SBSegment>();
```

Comment:
Apart from the loaded flag, they all can be 'private final'

Reply:
They can be or you want them to be?

Reply:
videoDuration also cannot be final (due to being assigned in loop). We could work around that, but thats just more ugly code to reach the same goal.